### ISSUE TITLE
GameTooltip_SetBackdropStyle LUA Error

### ISSUE THREAD

**Issue by waynemolina** [GoldpawsStuff/AzeriteUI#123]
### Make sure you can check all these boxes  

- [X] I have the latest addon version available from CurseForge.
- [X] I restarted the entire game after updating the addon, not just relogged or reloaded. (Only needed in Classic.)

### What game version(s) does it happen in? Choose your version and the highest expansion you own. 

- Live Realms
	- [X] Classic Era 
	- [ ] Burning Crusade Classic 
	- [ ] Shadowlands 
- Public Test Realm 
	- [ ] Classic Era 
	- [ ] Burning Crusade Classic 
	- [ ] Shadowlands 

### What addon theme(s) does it happen in?  

- [X] Azerite 
- [X] Legacy 
- [ ] I don't know what this means. 

### Does this happen with ONLY this addon enabled?  

- [X] Yes 
- [ ] No 

### Does this happen with absolutely all addons (this included) disabled?  

- [ ] Yes 
- [X] No 

### What classes does it happen on?  

- [ ] Death Knight 
- [ ] Demon Hunter 
- [ ] Druid 
- [ ] Hunter 
- [ ] Mage 
- [ ] Monk 
- [ ] Paladin 
- [ ] Priest 
- [ ] Rogue 
- [ ] Shaman 
- [X] Warlock 
- [ ] Warrior 

### Any logs, screenshots, error output, etc?  

This error upon logging in.  Note that player bar appears but action bars do not (stays Blizzard default)
```
Message: ...eUI\front-end\modules\blizzard\blizzard-tooltips.lua:833: hooksecurefunc(): GameTooltip_SetBackdropStyle is not a function
Time: Sun Nov 14 10:13:53 2021
Count: 1
Stack: ...eUI\front-end\modules\blizzard\blizzard-tooltips.lua:833: hooksecurefunc(): GameTooltip_SetBackdropStyle is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@Interface\AddOns\AzeriteUI\front-end\modules\blizzard\blizzard-tooltips.lua"]:833: in function `SetTooltipHooks'
[string "@Interface\AddOns\AzeriteUI\front-end\modules\blizzard\blizzard-tooltips.lua"]:918: in function `OnEnable'
[string "@Interface\AddOns\AzeriteUI\back-end\libraries\module.lua"]:282: in function `?'
[string "@Interface\AddOns\AzeriteUI\back-end\libraries\module.lua"]:737: in function `ForAll'
[string "@Interface\AddOns\AzeriteUI\back-end\libraries\module.lua"]:252: in function `Init'
[string "@Interface\AddOns\AzeriteUI\back-end\libraries\module.lua"]:867: in function `?'
[string "@Interface\AddOns\AzeriteUI\back-end\libraries\message.lua"]:65: in function `Fire'
[string "@Interface\AddOns\AzeriteUI\back-end\libraries\event.lua"]:71: in function <...erface\AddOns\AzeriteUI\back-end\libraries\event.lua:68>

Locals: 
```

Receive this error when trying to set up Actionbars from the options menu
```

### Steps to reproduce  
<!--If possible, tell us how to reproduce this issue. If you don't know exactly how to, please tell us what you were doing when it happened, in as much detail as possible. Include class, specialization, what quest/instance/battleground you were doing etc.-->

Install AzeriteUI 3.2.515-Release-classic from WoWUp

### Expected behaviour  
<!--Tell us what should happen.-->

No LUA error on logging in

### Actual behaviour  
<!--Tell us what happens instead.-->

See above LUA error

**Comment by goldpaw** [comment]
"Install AzeriteUI 3.2.515-Release-classic from **WoWUp**"

Or not? Use the official curseforge webpage, or the official curseforge app. Not third party clients robbing us of our profit and costing overwolf tons. Please.

**Comment by waynemolina** [comment]
Yeah, same errors with getting it from the webpage.  Not sure what the mode of download would have to do with the error...

**Comment by goldpaw** [comment]
Nothing. Never said it had. 

The method of reproducing the error is faulty, that was all. As WoWUp is a not a valid alternative to getting the addon. Once the most recent changes from Overwolf goes live, I will block access to all non-Curseforge/Overwolf methods of downloading any and all addons. 

Addons can be downloaded from github, curseforge webpage, curseforge app, or the wago page and apps. :+1:

**Comment by goldpaw** [comment]
Fixed in 3.2.516-Release
